content/bloggarkiv/2017-aret-da-gitops-kom-til-verden/index.md

```diff
@@ -0,0 +1,47 @@
+---
+# Publishing mode
+# draft = false means content is published. 
+draft: false
+# end
+
+title: "2017, året GitOps kom til verden"
+
+ingress: GitOps er eit rammeverk ein kan bruke i Continous Delivery, og blei nevnt for første gang i 2017 av Weaveworks. Same år fekk Digdir eigne erfaringar med lignande tankegang under etableringa av eInnsyn.
+image: /gitops-weave-works.png
+image_alt: "illustration"
+
+# Meta
+date: 2022-02-10T08:10:17+01:00
+readtime: 5 minutter
+---
+
+På vårparten i 2017 var ein av Digdir sine løysingar [einnsyn.no](https://einnsyn.no/) under etablering og ein såg på måtar å overlevere applikasjonar som skulle settast opp av drift. Grunna retningslinjer for tilgangstyring kunne ein ikkje la utvikling få direkte tilgang til containerplattformen for å rulle ut applikasjonar. Ein måtte finne ei løysing som kunne gi utvikling tilgang til å sette i gang utrulling inndirekte. eInnsyn er ei løysing basert på mikrotjeneste, og blei levert via Docker Compose filer.
```

Comment:
Hadde lagt til et par ord for å flytte siste setningen bedre inn. F.eks.:
"Ein måtte finne ei løysing som kunne gi utvikling tilgang til å sette i gang utrulling indirekte. Siden eInnsyn er ei løysing basert på mikrotjeneste, blei det levert via Docker Compose filer."